### PR TITLE
Add new attr of fused_multi_transformer

### PR DESCRIPTION
**Pull request by ZzSean**
### PR types

Function optimization
### PR changes

OPs
### Describe

Add new attr of fused_multi_transformer: `trans_qkvw`.
This attr is a flag to indicate whether the weights of qkv should be transposed. 
If true, the shape eights of qkv is `[3, num_head, dim_head, dim_embed]`.
Otherwise the shape of weights of qkv is `[dim_embed, 3, num_head, dim_head]`.

### CODE REVIEW

File: python/paddle/incubate/nn/functional/fused_transformer.py [hunk 1]
```diff
@@ -754,6 +755,7 @@ def fused_multi_transformer(x,
 
                                   - train: out = input * mask
                                   - inference: out = input * (1.0 - p)
+        trans_qkvw (bool, optional): Whether to transpose for weights of qkv. Default True.
```
Comment (qingqing01) [inline python/paddle/incubate/nn/functional/fused_transformer.py]: Need more detailed comments, like the commentis in C++ code

Comment (ZzSean) [reply python/paddle/incubate/nn/functional/fused_transformer.py]: done, thx

**Review by qingqing01 — APPROVED** [review]

**Review by Xreki — APPROVED** [review]
LGTM

**Review by lanxianghit — APPROVED** [review]

**Review by TCChenlong — APPROVED** [review]

**Review by jiweibo — APPROVED** [review]
LGTM for trans_qkvw attr (default value = true).

**Review by Shixiaowei02 — APPROVED** [review]

**Pull request merged**